.gitignore

```diff
@@ -4,3 +4,4 @@ linux/release
 build
 compile_commands.json
 thirdparty
+.vs
```

Comment:
The reason I've added this is that my Visual Studio always makes a `.vs` folder in the repository.